PostHog/Classes/PHGPayloadManager.m

```diff
@@ -157,6 +167,69 @@ - (void)alias:(NSString *)alias options:(NSDictionary *)options
                                options:options];
 }
 
+#pragma mark - Group
+
+- (void)group:(NSString *)groupType groupKey:(NSString *)groupKey groupProperties:(NSDictionary *)properties options:(NSDictionary *)options
+{
+    PHGGroupPayload *payload = [[PHGGroupPayload alloc] initWithType:groupType groupKey:groupKey
+                                                               properties:PHGCoerceDictionary(properties)];
+
+    [self callWithSelector:NSSelectorFromString(@"group:")
+                             arguments:@[ payload ]
+                               options:options];
+}
+
+#pragma mark - Feature Flags
+
+- (void)receivedFeatureFlags:(NSDictionary *)flags
+{
+    NSArray* keys = [flags allKeys];
+    
+#if TARGET_OS_TV
+        [self.userDefaultsStorage setArray:keys forKey:PHGActiveFeatureFlags];
+#else
+        [self.fileStorage setArray:keys forKey:kPHGActiveFeatureFlags];
+#endif
+    
+#if TARGET_OS_TV
+        [self.userDefaultsStorage setString:anonymousId forKey:PHGEnabledFeatureFlags];
```

Comment:
`setString:anonymousId` typo should be setArray:flags

Reply:
i have no idea why your tests that are failing are passing for me locally 🤣